CMakeLists.txt

```diff
@@ -176,31 +193,70 @@ foreach (_module ${HARDINFO_MODULES})
 	set_target_properties(${_module} PROPERTIES PREFIX "")
 endforeach()
 
-add_library(hardinfo-shell STATIC
-        shell/callbacks.c
-        shell/iconcache.c
-        shell/loadgraph.c
-        shell/menu.c
-        shell/report.c
-        shell/shell.c
-        shell/stock.c
+if (HARDINFO_GTK3)
```

Comment:
Can uber-graph be used in GTK2? If so, we can just get rid of the old loadgraph code and use it instead.

This should simplify the build system as well by removing some conditionals and copied lists of files.